src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="width=device-width,
+                 user-scalable=no,
+                initial-scale=1.0,
+                maximum-scale=1.0,
+                minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./styles/main.css"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body class="page">
+    <div class="card page__card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+
+      <p class="card__tittle1"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A) </p>
```

Comment:
If you have attributes and content - you can put content on separate line for better readability.
```suggestion
      <p class="card__tittle1">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </p>
```